week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -12,12 +12,19 @@
 
   Answer the following questions:
 
-  1. This program throws an error. Why? (If you can't find it, try executing it).
-  2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
+  1. This program throws an error. Why? (If you can't find it, try executing it). 'b' was not defined
+  2. Remove the line that throws the error. removed console.log(b)
+  3. What is printed to the console? 
+      2
+      6
+      4
+      9
+      6
+     13
+      8
+  4. How many times is "f1" called? 5
+  5. How many times is "f2" called? 5
```

Comment:
Please read the question again for this

Hint: f1 and f2 are not the same function